### PR TITLE
Fix missing video links not appearing

### PR DESCRIPTION
**Pull request by prince-chrismc**
### Bug

There's a video I wanted to share
https://github.com/conan-io/docs/blame/1c9f5384dadd1128deb75e9a985ca7045d608766/videos.rst?plain=1#L26

But it does not show up on the website 😖 

So I noticed this and was like maybe?! It's easy...

image

But nop! There's some magic white space thats missing

### Currently

I did not actually find the problem and I am kinda stuck :/

image

### CODE REVIEW

File: videos.rst [hunk 1]
```diff
@@ -14,35 +14,37 @@
...
 
   .. raw:: html
 
-      <iframe width="560" height="315" src="https://www.youtube.com/embed/kKGglzm5ous" frameborder="0" allow="autoplay; encrypted-media; allowfullscreen>
-      </iframe>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/NM-xp3tob2Q" frameborder="0" allow="autoplay; encrypted-media; allowfullscreen></iframe>
 
-      <br/><br/>
-      
+- There's a bug in the website where the second link is always removed ..........
```
Comment (prince-chrismc) [inline videos.rst:31]: This does not show up

File: videos.rst [hunk 2]
```diff
@@ -14,35 +14,37 @@
...
+
+- ACCU 2022: Advanced Dependencies Model in Conan 2.0 C, C++ Package Manager by Diego Rodriguez-Losada
 
   .. raw:: html
 
-      <iframe width="560" height="315" src="https://www.youtube.com/embed/vAe9k0NJkw8" frameborder="0" allow="autoplay; encrypted-media; allowfullscreen>
-      </iframe>
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/kKGglzm5ous" frameborder="0" allow="autoplay; encrypted-media; allowfullscreen></iframe>
+
+- Conan Package Manager for C++ in Practice
```
Comment (prince-chrismc) [inline videos.rst:43]: This is also missing

**Comment by prince-chrismc** [comment]
@uilianries you were last to play with these files, do you have any idea what's going wrong?

**Comment by uilianries** [comment]
@prince-chrismc no idea. I'll check on Monday. 😊

**Comment by prince-chrismc** [comment]
This is 1.x stuff

**Pull request closed without merging**